jbpm-bpmn2/src/test/resources/BPMN2-DataInputAssociations-Pojo.bpmn2

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="UTF-8"?> 
+<definitions id="Definition"
+             targetNamespace=""
+             typeLanguage="http://www.java.com/javaTypes"
+             expressionLanguage="http://www.mvel.org/2.0"
+             xmlns="http://www.omg.org/spec/BPMN/20100524/MODEL"
+             xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+             xsi:schemaLocation="http://www.omg.org/spec/BPMN/20100524/MODEL BPMN20.xsd"
+             xmlns:g="http://www.jboss.org/drools/flow/gpd"
+             xmlns:bpmndi="http://www.omg.org/spec/BPMN/20100524/DI"
+             xmlns:dc="http://www.omg.org/spec/DD/20100524/DC"
+             xmlns:di="http://www.omg.org/spec/DD/20100524/DI"
+             xmlns:tns="http://www.jboss.org/drools">
+
+  <itemDefinition id="_instanceMetadataItem" structureRef="org.jbpm.bpmn2.DataTest$Person" />
+
+  <process processType="Private" isExecutable="true" id="process" name="process" >
+  
+<!-- process variables -->
+    <property id="instanceMetadata" itemSubjectRef="_instanceMetadataItem"/>
+
+    <!-- nodes -->
+    <startEvent id="_1" name="" />
+    <userTask id="_2" name="Task">
+      <ioSpecification>
+        <dataInput id='coId' name='coId'/>
+        <inputSet>
+          <dataInputRefs>coId</dataInputRefs>
+        </inputSet>
+        <outputSet>
+        </outputSet>
+      </ioSpecification>
+      <dataInputAssociation>
+        <sourceRef>instanceMetadata</sourceRef>
+        <targetRef>coId</targetRef>
+        <assignment>
+          <from language="http://www.mvel.org/2.0">#{address.city}</from>
```

Comment:
Could you add a comment indicating that language attribute validation against xsd will fail here (though it shouldn't, according to the standard, as you mention before)?
It's just to keep in mind in the future that this is a known issue to us in the xsd definition.

Reply:
This is a typo and should not be there.